package.json

```diff
@@ -147,31 +147,35 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "webpack --mode production && cp -rf node_modules/@influxdata/flux-lsp-cli/out dist/.",
+    "copy-wasm": "rm -f dist/flux-lsp-node_bg.wasm && cp node_modules/@influxdata/flux-lsp-node/flux-lsp-node_bg.wasm dist/",
+    "vscode:prepublish": "npm run webpack-prod && npm run copy-wasm",
```

Comment:
when i use f5, seems my webpack dev mode is broken.

Reply:
there is a script to copy this in npm script, its called in the prepublish script

Reply:
do you need to run `copy-wasm` for dev mode as well?